Symfony/CS/Tests/Fixer/Contrib/ShortBoolCastFixerTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Contrib;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+class ShortBoolCastFixerTest  extends AbstractFixerTestBase
```

Comment:
yea I got so tired of doing wrong that I started on that fixer ^-^ 
fixed and move to Symfony level